src/helpers/redact-strings-in-log-args.helper.ts

```diff
@@ -1,29 +1,31 @@
 import { get, set } from 'lodash';
 
 // This helper function is used to redact sensitive data in Error object strings.
-export const redactError = (
+export const redactStringsInLogArgs = (
   redactLogs: boolean,
   redactPaths: string[],
   redactStrings: { searchValue: string | RegExp; replaceValue: string }[],
-  err: any,
+  args: any,
```

Comment:
Can we put any stricter type that `any` here? Do we know it's an array at least?

Reply:
I looked at pico source code and documentation, looks like type is universal, only limit is that it is an array. Changed to any[]